### DISCUSSION TITLE
希望增加一个颜色应用库

### DISCUSSION THREAD

**Discussion by DiXuanAoYi** [WPFDevelopersOrg/WPFDevelopers#13] [Ideas]
wpf公共库麻烦再加一个颜色应用库。
提供标准颜色器功能
01———识色
![image](https://user-images.githubusercontent.com/39654103/196084004-302532c4-84e6-4f0d-8e2e-b38bf4672585.png)

02———颜色转换RGB HSL HSV等

03———其他的对比色，互补色，临近色等在色轮上辅助显示。
![image](https://user-images.githubusercontent.com/39654103/196084081-5d92470f-d06c-4a4b-b322-087da85c2ff9.png)
![image](https://user-images.githubusercontent.com/39654103/196084184-c0e2a23b-c38f-4afb-82f7-2574228c8f4e.png)

04———提供图象画面颜色面积计算。

05———提供分析画面颜色或无色图在0.0.0-1.1.1之间的分布遮照显示

06———允许图像颜色整体或两端重新映射到指定区间30.30.30-240.240.240生成新图

07———图像颜色检查整体在自定义区间30.30.30-240.240.240遮罩标示哪些地方超出了区间，区间要能自定义最好别写死

**Comment by yanjinhuagood** [comment]
@DiXuanAoYi https://github.com/WPFDevelopersOrg/WPFDevelopers/tree/master/src/WPFDevelopers.Shared/Controls/ColorPicker
